app/src/main/java/com/developersbreach/composeactors/ui/screens/favorites/tabs/FavoriteMoviesTabContent.kt

```diff
@@ -25,36 +32,61 @@ import com.developersbreach.composeactors.ui.theme.ComposeActorsTheme
  * Content shown for home tab Favorites.
  * If user did not add any movies to favorites, a message will be shown.
  */
+@OptIn(ExperimentalFoundationApi::class)
 @Composable
 fun FavoriteMoviesTabContent(
     getSelectedMovieDetails: (Int) -> Unit,
     favoriteMovies: List<Movie>,
+    removeFavoriteMovie: (Movie) -> Unit,
 ) {
     if (favoriteMovies.isEmpty()) {
         NoFavoritesFoundUI()
     }
 
-    LazyVerticalGrid(
-        columns = GridCells.Fixed(3),
-        modifier = Modifier.fillMaxWidth(),
-        horizontalArrangement = Arrangement.spacedBy(12.dp),
-        verticalArrangement = Arrangement.spacedBy(12.dp),
-        contentPadding = PaddingValues(vertical = 2.dp, horizontal = 16.dp)
+    LazyColumn( modifier = Modifier
+        .fillMaxWidth()
     ) {
-        items(
-            items = favoriteMovies,
-            key = { it.movieId }
-        ) { movieItem ->
-            LoadNetworkImage(
-                imageUrl = movieItem.posterPathUrl,
-                contentDescription = stringResource(R.string.cd_movie_poster),
-                shape = MaterialTheme.shapes.medium,
+        items(favoriteMovies.size) { index ->
```

Comment:
We should not do this. 
To get the reference of any movie item, you can use `LazyListScope.items` .
```kotlin
    import androidx.compose.foundation.lazy.items

    LazyColumn(
        modifier = Modifier.fillMaxWidth()
    ) {
        items(favoriteMovies) { movie: Movie ->
            Box {
                LoadNetworkImage(
                    imageUrl = movie.posterPathUrl,
                            getSelectedMovieDetails(movie.movieId)
                )
                Text(
                    text = movie.movieName,
                )
                IconButton(onClick = { removeFavoriteMovie(movie) }
            }
        }
    }
```